fast64_internal/f3d/f3d_gbi.py

```diff
@@ -1787,7 +1790,9 @@ def get_tex_sts_code(tex: FSetTileSizeScrollField, tex_num: int, cmd_num: int):
 def mat_tile_scroll(
     mat: str, tex0: FSetTileSizeScrollField, tex1: FSetTileSizeScrollField, cmd_num0: int, cmd_num1: int
 ):
-    func = f"void scroll_sts_{mat}()"
+    # remove any offsets from mat name
+    func_name = mat.split("+")[0].strip()
+    func = f"void scroll_sts_{func_name}()"
```

Comment:
This is a bit hacky, also related to tile scrolling bleeds if I'm understanding correctly.